tests/test_bndfun.py

```diff
@@ -95,72 +96,79 @@ def test_call_clenshaw(self):
 
     def test_copy(self):
         ff = self.ff
         gg = self.ff.copy()
         self.assertEquals(ff, ff)
         self.assertEquals(gg, gg)
         self.assertNotEquals(ff, gg)
-        self.assertEquals(infnorm(ff.coeffs-gg.coeffs), 0)
+        self.assertEquals(infnorm(ff.coeffs - gg.coeffs), 0)
 
     # check that the restricted fun matches self on the subinterval
     def test_restrict(self):
-        i1 = Interval(-1,1)
+        i1 = Interval(-1, 1)
         gg = self.ff.restrict(i1)
-        yy = np.linspace(-1,1,1000)
-        self.assertLessEqual(infnorm(self.ff(yy)-gg(yy)), 1e2*eps)
+        yy = np.linspace(-1, 1, 1000)
+        self.assertLessEqual(infnorm(self.ff(yy) - gg(yy)), 1e2 * eps)
 
     # check that the restricted fun matches self on the subinterval
     def test_simplify(self):
-        interval = Interval(-2,1)
+        interval = Interval(-2, 1)
         ff = Bndfun.initfun_fixedlen(self.f, interval, 1000)
         gg = ff.simplify()
         self.assertEqual(gg.size, standard_chop(ff.onefun.coeffs))
-        self.assertEqual(infnorm(ff.coeffs[:gg.size]-gg.coeffs), 0)
+        self.assertEqual(infnorm(ff.coeffs[: gg.size] - gg.coeffs), 0)
         self.assertEqual(ff.interval, gg.interval)
 
     def test_translate(self):
         c = -1
         shifted_interval = self.ff.interval + c
         gg = self.ff.translate(c)
-        hh = Bndfun.initfun_fixedlen(lambda x: self.ff(x-c), shifted_interval, gg.size)
-        yk = shifted_interval(np.linspace(-1,1,100))
+        hh = Bndfun.initfun_fixedlen(
+            lambda x: self.ff(x - c), shifted_interval, gg.size
+        )
+        yk = shifted_interval(np.linspace(-1, 1, 100))
         self.assertEqual(gg.interval, hh.interval)
-        self.assertLessEqual(infnorm(gg.coeffs-hh.coeffs), 2e1*eps)
-        self.assertLessEqual(infnorm(gg(yk)-hh(yk)), 1e2*eps)
+        self.assertLessEqual(infnorm(gg.coeffs - hh.coeffs), 2e1 * eps)
+        self.assertLessEqual(infnorm(gg(yk) - hh(yk)), 1e2 * eps)
+
 
 # --------------------------------------
 #          vscale estimates
 # --------------------------------------
 vscales = [
     # (function, number of points, vscale)
-    (lambda x: sin(4*pi*x),      [-2, 2],     1),
-    (lambda x: cos(x),           [-10, 1],    1),
-    (lambda x: cos(4*pi*x),      [-100, 100], 1),
-    (lambda x: exp(cos(4*pi*x)), [-1,1],      exp(1)),
-    (lambda x: cos(3244*x),      [-2,0],      1),
-    (lambda x: exp(x),           [-1,2],      exp(2)),
-    (lambda x: 1e10*exp(x),      [-1,1],      1e10*exp(1)),
-    (lambda x: 0*x+1.,           [-1e5,1e4],  1),
+    (lambda x: sin(4 * pi * x), [-2, 2], 1),
+    (lambda x: cos(x), [-10, 1], 1),
+    (lambda x: cos(4 * pi * x), [-100, 100], 1),
+    (lambda x: exp(cos(4 * pi * x)), [-1, 1], exp(1)),
+    (lambda x: cos(3244 * x), [-2, 0], 1),
+    (lambda x: exp(x), [-1, 2], exp(2)),
+    (lambda x: 1e10 * exp(x), [-1, 1], 1e10 * exp(1)),
+    (lambda x: 0 * x + 1.0, [-1e5, 1e4], 1),
```

Comment:
This is one example of where the auto-formatter makes the set of test functions hard to read